src/index.html

```diff
@@ -2,11 +2,428 @@
+              data-qa="hover"
+              href="#buy"
+            >
+              BUY
+            </a>
+          </div>
+        </div>
+        <div class="card">
+          <img
+            class="card__img"
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+          >
+          <div class="card__description">
+            <div>
+              <h2 class="card__name">
+                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+              </h2>
+              <p class="card__product-code">
+                Product code: 195434
+              </p>
+            </div>
+
+            <div class="card__reviews">
+              <div class="card__stars stars stars--4">
+                <div class="stars__item star"></div>
+                <div class="stars__item star"></div>
+                <div class="stars__item star"></div>
+                <div class="stars__item star"></div>
+                <div class="stars__item star"></div>
+              </div>
+              <div>Reviews: 5</div>
+            </div>
+
+            <div class="card__price">
+              <div>
+                Price:
+              </div>
+              <div class="card__price-current">
+                $2,199
+              </div>
+            </div>
+            <a
+              class="card__buy-button"
+              data-qa="hover"
+              href="#buy"
+            >
+              BUY
+            </a>
+          </div>
+        </div>
+        <div class="card">
+          <img
+            class="card__img"
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+          >
+          <div class="card__description">
+            <div>
+              <h2 class="card__name">
+                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+              </h2>
+              <p class="card__product-code">
+                Product code: 195434
+              </p>
+            </div>
+
+            <div class="card__reviews">
+              <div class="card__stars stars stars--4">
+                <div class="stars__item star"></div>
+                <div class="stars__item star"></div>
+                <div class="stars__item star"></div>
+                <div class="stars__item star"></div>
+                <div class="stars__item star"></div>
+              </div>
+              <div>Reviews: 5</div>
+            </div>
+
+            <div class="card__price">
+              <div>
+                Price:
+              </div>
+              <div class="card__price-current">
+                $2,199
+              </div>
+            </div>
+            <a
+              class="card__buy-button"
+              data-qa="hover"
+              href="#buy"
+            >
+              BUY
+            </a>
+          </div>
+        </div>
+      </div>
+
+    </main>
   </body>
-</html>
+</html>
```

Comment:
Newline is required at the end
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file